pages/web-sites.md

```diff
@@ -120,8 +120,14 @@ For other sites:
 
 To create a tag, use `npm version [major | minor | patch]`.
 
-Afterwards, make sure to push both version change commit and the tag:
-`git push --tags origin master`
+Afterwards, make sure to push both version change commit and the tag to the main jQuery repo.
+
+<div class="warning">You'll want to check your remote setup using <code>git remote -v</code> and check which remote is used for the main jQuery repo.</div>
+
+If `upstream` is the remote you use for that, type:
+`git push --tags upstream master`
+
+If you use `origin` for the main jQuery repo, replace `upstream` with `origin` in the command above.
```

Comment:
I probably should've read the whole thing before I started commenting. Perhaps take my suggested wording and move it down here, replacing "below" and "above" and adding this sentence. That would give you:

> The example above uses `upstream` for the repo in the jQuery organization on GitHub. Use `git remote -v` to ensure that you're pushing to the correct remote repo. For example, If you use `origin` for the repo in the jQuery organization, replace `upstream` with `origin`.